packages/react-keytips/.storybook/manager.ts

```diff
@@ -0,0 +1,5 @@
+import { addons } from '@storybook/manager-api';
+
+addons.setConfig({
+  enableShortcuts: false,
```

Comment:
disabled the shortcuts, as some of them were conflicting with keytips hotkeys. For instance, if `invokeEvent` is `keyup`, digit 1 is not captured at all.